scripts/scraper-ng/index.js

```diff
@@ -0,0 +1,58 @@
+const rehype = require("rehype");
+const reporter = require("vfile-reporter");
+
+const toVFile = require("./url-to-vfile");
+
+const examplePage = "https://developer.mozilla.org/…/div";
+const examplePath = "https://developer.mozilla.org/…/div";
```

Comment:
examplePath and examplePage are the same. Were you intending to support pages and page trees to scrape?
Also, this isn't a real URL, would it be better to use real examples?

Reply:
Yeah, an artifact of my development "process" 😆

I originally thought I'd allow you to supply a bunch of URLs at once. That's… complicated. I'm going to pare this back now, but we'll definitely need a bulk option. More on this in a bit.

Reply:
Oh wait, I misunderstood this. Originally, I was going to let you supply a full URL or a shorthand path (e.g., `https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div` or `/en-US/docs/Web/HTML/Element/div`). Not doing that at all.

Reply:
OK, I've fixed the help text and the code to reflect what's actually happening with one URL at a time. I'll address the page trees question in a comment later.